redisinsight/ui/src/services/workbenchStorage.ts

```diff
@@ -0,0 +1,350 @@
+import { reverse } from 'lodash'
+import { CommandExecution } from 'uiSrc/slices/interfaces'
+import { BrowserStorageItem } from 'uiSrc/constants'
+import { CommandExecutionStatus } from 'uiSrc/slices/interfaces/cli'
+import { getConfig } from 'uiSrc/config'
+import { formatBytes } from 'uiSrc/utils'
+
+const riConfig = getConfig()
+
+export class WorkbenchStorage {
+  private db?: IDBDatabase
+
+  constructor(private readonly dbName: string, private readonly version = 1, private readonly keySeparator = ':') {
+  }
+
+  private initDb(storeName: string) {
+    return new Promise((resolve, reject) => {
+      if (!window.indexedDB) {
+        reject(new Error('indexedDB is not supported'))
```

Comment:
I believe you need a return after this, otherwise script execution will continue